input/full.go

```diff
@@ -81,6 +81,12 @@ func CollectFull(server *state.Server, connection *sql.DB, globalCollectionOpts
 		err = nil
 	}
 
+	ts.ServerStats, err = postgres.GetServerStats(logger, connection, ts.Version, systemType)
+	if err != nil {
+		logger.PrintWarning("Error collecting Postgres server statistics: %s", err)
+		return
```

Comment:
If we consider it a warning, we should probably drop the return (so the snapshot can be finished and sent) - otherwise we should make it an error to match existing code.

The downside of outputting a log message here but continuing is that you will also get a successful test run. If the error is very unlikely its probably best to use PrintError here.